### ISSUE TITLE
Investigate butterfly data extract from Silvia

### ISSUE THREAD

**Issue by rukayaj** [SANBIBiodiversityforLife/species#98]
[NewAssessDB_Dewidine example.xlsx](https://github.com/rukayaj/species/files/1196888/NewAssessDB_Dewidine.example.xlsx)

Attached is an example of what our assessment data extract looks like - the final export will look like this. Some of the fields aren't applicable for this assessment.

We will also have separate csv files for the various tables (e.g. threats, bibliography, veg types, etc.), as per the plant example sent to us by Lize a long time ago. These are not yet ready.

**Comment by Dewidine** [comment]
@rukayaj @DomitillaRaimondo We need to urgently figure this out, the butterflies will be ready soon and all the data needs to be uploaded. I will check with them, whether they have any of the csv's that we sent them, ready for export. This spreadsheet is different from the SIS exports, so a new import script might be needed here... So time needs to be spent on understandings this task.

**Comment by Dewidine** [comment]
@rukayaj 
[NewAssessDB_andre_absolute final.xlsx](https://github.com/SANBIBiodiversityforLife/species/files/1719506/NewAssessDB_andre_absolute.final.xlsx)
Update of export

**Comment by rukayaj** [comment]
Ok it's possible to import this apart from the references which is going to be a headache. This is an example of one of their references:
"Mecenero S., Ball J.B., Edge D.A., Hamer M.L., Henning G.A., Kruger M., Pringle E.L., Terblanche R.F., Williams M.C.: , 2013; Conservation Assessment of Butterflies of South Africa, Lesotho and Swaziland: Red List and Atlas"
What I really need is the bibtex of these OR for the reference to be in the same format as SIS, where each part of the reference is listed separately. Who do they have doing the development work? Perhaps might be worth showing them https://github.com/AusterweilLab/bibtex-js-apa and similar http://okfnlabs.org/bibjson/ which can get stored + searched in any database which has jsonb fields. I really feel like this is the way to go when it comes to storing bibliographic data. 

Just want to check before I start writing this script (which will probably take me a couple of days) that they aren't going to have it in SIS export format any time soon? If so then might be worthwhile us waiting for that...

**Comment by Dewidine** [comment]
@rukayaj I'll send your suggestion for their references to their guy and cc you in, if he has any further technical questions, he can communicate with you. Maybe we need a Skype, but lets see. Just to confirm, the import from the spreadsheet is fine, its only the references that will need a different method? I'll keep you in the loop.

**Comment by rukayaj** [comment]
Yeah the import from the spreadsheet is fine - by fine i mean a few hours to write an import script to handle it. If we had their SIS extract instead I could import it immediately like I did with the Sparids. If they're not going to get that sorted though I had better get started writing that script.

**Comment by Dewidine** [comment]
@rukayaj This is an update of a national assessment so we need to apply versioning, we nee to discuss this urgently, Tilla will try and get some of your time, while you are in Pretoria to discuss it.

**Comment by rukayaj** [comment]
@Dewidine I didn't see Tilla so you and I will have to chat about this in cape town. When are you free? I have broken the reports on opus.sanbi.org and Ferozah needs them fixed asap, so need to work on that today. I'm hoping I will be done by the end of the day if I concentrate solidly on them.

**Comment by rukayaj** [comment]
Ok there's a bit of confusion with Silvia having a problem with conservation actions, but apart from that I can just wait for their SIS extract
